### DISCUSSION TITLE
I'd like to contribute some code to Presto but I don't know whom to ask for a code review :)

### DISCUSSION THREAD

**Discussion by larrydiamond** [prestodb/presto#15368] [Q&A]
https://github.com/prestodb/presto/pull/15367/files

Some minor league performance boosts, it's passed the tests from Travis CI.

I just don't know whom to ask for a review :)

First time code contributor to Prestodb - I've contributed elsewhere.

Thank you very much
Larry Diamond

**Comment by rschlussel** [comment]
Just reviewed your commit.  In the future feel free to ask any of the committers and if we're not the right person to review, we'll route it to the right person. https://github.com/prestodb/presto/wiki/committers.
